tools/swift-demangle-fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_swift_host_tool(swift-demangle-fuzzer
+  swift-demangle-fuzzer.cpp
+  LINK_LIBRARIES swiftDemangling
+  LLVM_COMPONENT_DEPENDS support
+  SWIFT_COMPONENT compiler
+  )
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=fuzzer")
```

Comment:
Could you instead of doing this create a cmake wrapper like.

add_swift_fuzzing_host_tool or something like that.